examples/js/app.ts

```diff
@@ -40,17 +40,19 @@ autocomplete<Hit<QuerySuggestionHit>>({
     }).then(([hits]) => {
       return [
         {
-          getInputValue: ({ suggestion }) => suggestion.query,
-          getSuggestions() {
+          getItemInputValue({ item }) {
+            return item.query;
+          },
+          getItems() {
             return hits;
           },
           templates: {
             item({ item }) {
               return `
                 <div class="item-icon">${searchIcon}</div>
                 <div>
-                  ${reverseHighlightItem({
-                    item,
+                  ${reverseHighlightHit({
```

Comment:
what do we win by calling this a hit, while we talk about item in the other spots? Is it because it's called getAlgoliaHits? Could that maybe be called `getAlgoliaItems` or `search` to remove the word hits?

Reply:
An item is a generic object manipulated by Autocomplete. A hit is an item transformed by Algolia which contains `_highlightResult`, etc. A hit is a search term and it reflects the Algolia brand.